.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.0.1
+current_version = 0.3.0
```

Comment:
I think we can leave it as 0.3.0 for now because that's the version of the last CLI release.  This will become irrelevant once we move the CLI into the CARROT repo at the next release.

Reply:
Sounds good.